README.md

```diff
@@ -1,48 +1,42 @@
-
-- [Getting Started](#getting-started)
-    - [Running the Design System Docs](#running-the-design-system-docs)
-- [Packages](#packages)
-    - [Monorepo](#monorepo)
-    - [Versioning](#versioning)
-    - [Development](#development)
-        - [Publishing a new version](#publishing-a-new-version)
-
-<!-- /TOC -->
+## Packages
 
-## Getting started
+| Package/README                                                                                                        | Description                                                              |
+| --------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| [@okta/odyssey-scss](https://github.com/okta/odyssey/tree/master/packages/odyssey-scss/README.md)                               | Odyssey SCSS, includes components, includes base and reset styles.       |
+| [@okta/odyssey-icons](https://github.com/okta/odyssey/tree/master/packages/odyssey-icons/README.md)                   | Odyssey SVG icons.                                                       |
+| [@okta/odyssey-scss](https://github.com/okta/odyssey/tree/master/packages/odyssey-scss/README.md) | Odyssey SCSS used across the system |
+| [@okta/odyssey-react](https://github.com/okta/odyssey/tree/master/packages/odyssey-react/README.md)                   | Odyssey components for [React](https://reactjs.org/).                 |
 
-We use [Yarn](https://github.com/yarnpkg/yarn) as our node package manager client. To install Yarn, check out their [installation instructions](https://yarnpkg.com/getting-started/install).
+## Git Workflow
+This project takes care to use standard gitflow workflow with minor exceptions.
 
-```bash
-# Clone this repo and navigate into it
+### Branches
+- `master` contains code for the latest version of Odyssey. The code in here is tagged with a semver and release notes. This is a protected branch.
+- `develop` is a working branch which contains the latest working code from different branches. This is a protected branch.
+- `feature/${branch-name}` the feature prefix is reserved for new feature work
+- `bugfix/${branch-name}` the bugfix prefix is reserved for existing features that have bugfixes
 
-git clone git@github.com:okta/odyssey.git
-cd odyssey
+Additionally, `refactor` and `hotfix` branch prefixes may be necessary in certain instances. Use at your discression.
 
-# Install all package dependencies
-yarn install
 ```
 
-### Running the design system docs
-
-Want to see Odyssey in action? Our `docs` package contains a static website styled using Odyssey. Simply run the following command to launch it:
+           -▸ develop --▸ feature/branch-name --▸ | --▸ commit #1
+                 ▲                                | --▸ commit #2
+                 |                                | --▸ commit #3
+  Squash & Merge |                                | --▸ commit #4
```

Comment:
I'm used to doing this because it's not uncommon for a working branch to have a bunch of commits that may not be useful. I would think that squashing and merging into develop can only assist with things like conventional commits and keeping the history tidy?

Reply:
Will Remove